h/static/scripts/vendor/jquery.scrollintoview.js

```diff
@@ -109,29 +109,37 @@
 
 				var animOptions = {};
 
+				var yDir = "none";
```

Comment:
> > Scrolling [...] and we've never fully implemented it to deal with all the cases that present.

> Which cases are not dealt with?

There is one limitation that I am aware of: we don't fully handle situations where the scrolling happens at a child element (instead of the main document), or when we have two, independently scrollable elements on the same document. (See #821.)

Our whole bucket bar design doesn't fully cover those cases. However, it's not only the "scrolling" which is problematic in those situations; the whole thing needs a full rethinking. So it's not like our problems (with those outlier cases) would go away if we removed our ability to scroll the document to locate the anchors.